packages/core/tsconfig.cjs.json

```diff
@@ -1,6 +1,5 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["./src"],
```

Comment:
This is already included in the `tsconfig.json` file we extend.

Reply:
This is what make the CI fail. Those files are auto-generated by a script, so you'd need to modify that script in `/tools/`.

But it doesn't really matter, so let's just keep this out of this PR.